dm/master/shardddl/pessimist.go

```diff
@@ -146,6 +155,110 @@ func (p *Pessimist) Locks() map[string]*pessimism.Lock {
 	return p.lk.Locks()
 }
 
+// UnlockLock unlocks a shard DDL lock manually when using `unlock-ddl-lock` command.
+// ID: the shard DDL lock ID.
+// replaceOwner: the new owner used to replace the original DDL for executing DDL to downstream.
+//   if the original owner is still exist, we should NOT specify any replaceOwner.
```

Comment:
in which case the original owner is not existing?

Reply:
and maybe we should choose a `replaceOwner` automatic later.

Reply:
> in which case the original owner is not existing

`operate-source remove` takes on the original owner.

> and maybe we should choose a replaceOwner automatic later.

YES, it's better to do that automatically.